packages/expo-splash-screen-command/README.md

```diff
@@ -0,0 +1,60 @@
+# expo-splash-screen-command
+
+This module provides supporting installation command for [`expo-splash-screen`](https://github.com/expo/expo/tree/master/packages/expo-splash-screen) module.
+You can use this command to configure your native iOS and Android project according to your needs without opening XCode or Android Studio.
+
+## Content
+
+- [📜	CHANGELOG](./CHANGELOG.md)
+- [🚀 Features](#-features)
+- [🗒 Usage](#-usage)
+- [🖥 Installation](#-installation)
+- [👏 Contributing](#-contributing)
+
+## 🚀 Features
+
+### 📱 iOS
+
+- configures background color for native splash screen
+- configures [`expo-splash-screen`](https://github.com/expo/expo/tree/master/packages/expo-splash-screen) to show given `.png` image
+- supports [`CONTAIN`](https://github.com/expo/expo/tree/master/packages/expo-splash-screen#contain-resize-mode) and [`COVER`](https://github.com/expo/expo/tree/master/packages/expo-splash-screen#cover-resize-mode) modes from [`expo-splash-screen`](https://github.com/expo/expo/tree/master/packages/expo-splash-screen)
+
+### 🤖 Android
+
+- configures background color for native splash screen
+- configures `expo-splash-screen` to show given `.png` image
+- supports [`CONTAIN`](https://github.com/expo/expo/tree/master/packages/expo-splash-screen#contain-resize-mode), [`COVER`](https://github.com/expo/expo/tree/master/packages/expo-splash-screen#cover-resize-mode) and [`NATIVE`](https://github.com/expo/expo/tree/master/packages/expo-splash-screen#native-resize-mode) modes from [`expo-splash-screen`](https://github.com/expo/expo/tree/master/packages/expo-splash-screen)
+
+## 🗒 Usage
+
+Command syntax:
+```
+yarn run expo-splash-screen --mode "contain" (default) | "cover" | "native" (only on Android) --platform "all" (default) | "ios" | "android" <backgroundColor> (required) [imagePath] (optional)
```

Comment:
😮 😮 
It's so messy and might be very hard to read and understand unfortunately... Maybe just:
```suggestion
yarn run expo-splash-screen --mode <mode> --platform "all" <backgroundColor> [imagePath]
```
and explain below what those options and arguments do, what's the default value and possible options